pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[tool.poetry]
+name = "schematicpy"
+version = "0.1.11"
+description = "Package for data model and data ingress management"
+authors = [
+    "Milen Nikolov <milen.nikolov@sagebase.org>",
+    "Sujay Patil <sujay.patil@sagebase.org>",
+    "Bruno Grande <bruno.grande@sagebase.org>",
+    "Xengie Doan <xengie.doan@sagebase.org>"
+]
+
+readme = "README.md"
+
+homepage = "https://github.com/Sage-Bionetworks/schematic"
+repository = "https://github.com/Sage-Bionetworks/schematic"
+documentation = "https://github.com/Sage-Bionetworks/schematic"
+
+keywords = ["schema", "metadata", "validation", "data model", "linked data"]
+
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "Topic :: Software Development :: Libraries :: Python Modules"
+]
+
+packages = [
+    { include = "schematic" }
+]
+
+[tool.poetry.scripts]
+schematic = "schematic.__main__:main"
+
+
+[tool.poetry.dependencies]
+python = "^3.7.1"
+
+click = "^7.1.2"
+click-log = "^0.3.2"
+google-api-python-client = "^1.12.8"
+google-auth-httplib2 = "^0.0.4"
+google-auth-oauthlib = "^0.4.2"
+graphviz = "^0.16"
+importlib-metadata = {version = "^1.6.0", python = "<3.8"}
+inflection = "^0.5.1"
+jsonschema = "^3.2.0"
+networkx = "^2.5"
+oauth2client = "<4.0.0"  # Specified because of bug in version ^4.0.0
+orderedset = "^2.0.3"
+pandas = "^1.2.1"
+pygsheets = "^2.0.4"
+PyYAML = "^5.4.1"
+rdflib = "^5.0.0"
+setuptools = "^52.0.0"
+synapseclient = "^2.2.2"
+tabletext = "^0.1"
```

Comment:
You can remove this import statement from here: [`import tabletext`](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/schemas/explorer.py#L7), and also this print statement from here: [`print(tabletext.to_text(content))`](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/schemas/explorer.py#L263). I suspect we might do something different for logging purposes, so we can add a different logging statement when dealing with issue #317.

Reply:
Thanks!